drivers/windows/R/driver.R

```diff
@@ -14,15 +14,15 @@ hipercow_driver_windows <- function() {
 }
 
 
-windows_submit <- function(id, config, path_root) {
+windows_submit <- function(id, resources, config, path_root) {
   path_batch <- write_batch_task_run(id, config, path_root)
 
   path_batch_dat <- prepare_path(path_batch, config$shares)
   path_batch_unc <- windows_path_slashes(
     file.path(path_batch_dat$path_remote, path_batch_dat$rel))
 
   client <- get_web_client()
-  dide_id <- client$submit(path_batch_unc, id, config$template)
+  dide_id <- client$submit(path_batch_unc, id, resources)
```

Comment:
Can you remove template through the configuration of the driver? Or just make a ticket and we can do it later

Reply:
Think this is now done